opds2_import.py

```diff
@@ -823,7 +823,8 @@ def extract_last_update_dates(self, feed):
 
         return dates
 
-    def extract_feed_data(self, feed, feed_url=None, custom_identifier=None):
+    def extract_feed_data(self, feed, feed_url=None,
+                          primary_identifier_source=None):
```

Comment:
The final argument is no longer used in this method; take it out altogether to avoid changing the method signature.